code/vocabulary.py

```diff
@@ -159,3 +161,54 @@ def replace_words(words: Sequence[str]) -> Sequence[str]:
             # fall back to default implementation for error-robustness
             logging.error("phrase replacer failed!")
             return actions.next(words)
+
+def _create_vocabulary_entries(spoken_form, written_form, type):
+    """Expands the provided spoken form and written form into multiple variants based on
+    the provided type, which can be either "name" to add a possessive variant or "noun"
+    to add plural.
+    """
+    entries = {spoken_form: written_form}
+    if type == "name":
+        entries[f"{spoken_form}s"] = f"{written_form}'s"
+    elif type == "noun":
+        entries[f"{spoken_form}s"] = f"{written_form}s"
+    return entries
```

Comment:
Consider vendoring/incorporating https://github.com/jpvanhal/inflection/blob/master/inflection/__init__.py ?

Reply:
I'm open to it, but I want to point out that this feature is primarily used for custom vocabulary words, in which case I'm not sure that normal English rules are as likely to apply, so there is some benefit to the predictability of my approach here. For example, "equipment" is listed as uncountable, but if this were a class name in a programming language I could imagine that "equipments" would be appropriate. Of course there are plenty of contrary examples such as nouns ending in "y" where you'd probably want "ies" -- my point is just that there may be some benefit to the predictability. But like I said, I definitely don't feel strongly and would be happy to add this file with attribution and do a relative import.

Reply:
I don't feel strongly about this; perhaps we can see how people use this in practice. May help to add some comments in vocabulary.talon, perhaps discussing your intended use case and/or examples?